moodle/Sniffs/Files/BoilerplateCommentSniff.php

```diff
@@ -233,8 +233,16 @@ private function fullComment(): array
 
     private function insertBoilerplate(File $file, int $stackptr): void
     {
-        $prefix = substr($file->getTokens()[$stackptr]['content'], -1) === "\n" ? '' : "\n";
-        $file->fixer->addContent($stackptr, $prefix . implode("\n", $this->fullComment()) . "\n");
+        $token = $file->getTokens()[$stackptr];
+        $paddedComment = implode("\n", $this->fullComment()) . "\n";
+
+        if ($token['code'] === T_OPEN_TAG) {
+            $replacement = trim($token['content']) . "\n" . $paddedComment;
+            $file->fixer->replaceToken($stackptr, $replacement);
+        } else {
```

Comment:
Hi @micaherne ,

I've just seen that the new code now leaves these 2 few lines uncovered, when they were previously covered? Can you ensure that the uncovered code can happen and, because of that, it should have some fixture covering it?

Ciao :-)

Reply:
Or, with other words... why the existing fixtures (before this patch), don't cover those lines when they previously were? I mean, is "smells" that they have become uncovered with your patch.

Reply:
Thanks for the feedback @stronk7! I wasn't sure what to do about the code coverage but I added a test file with a phpcs tag and it's causing that branch to run.